core/test/dataset_test_common.h

```diff
@@ -37,13 +37,16 @@ class DatasetFactory3D {
   DatasetFactory3D(const scipp::index lx = 4, const scipp::index ly = 5,
                    const scipp::index lz = 6);
 
+  const Dataset &empty() const { return base; }
   Dataset make();
 
-private:
+public:
```

Comment:
Can't see where this is required...?

Reply:
Yes, these changes are no longer needed.